backend/pkg/api/routes/users.go

```diff
@@ -0,0 +1,58 @@
+package routes
+
+import (
+	"encoding/json"
+	"github.com/julienschmidt/httprouter"
+	"net/http"
+	"raedar/pkg/api/responses"
+)
+
+type userRegisterRequest struct {
+	Email    string `json:"email"`
+	Username string `json:"username"`
+	Password string `json:"password"`
+}
+
+// Register registers a new user into the system.
+func Register() httprouter.Handle {
+	return func(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+		w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+		w.WriteHeader(http.StatusOK)
+
+		response := &responses.JSONSuccess{Data: "Welcome to raedar", Success: true}
+		if err := json.NewEncoder(w).Encode(response); err != nil {
+			panic(err)
+		}
+	}
+}
+
+// // UserRoutes creates all user routes
```

Comment:
Oh thanks for pointing this out @huxaiphaer , I am going to remove this commented code in the later code because I will only need it for a few set ups thats why I let it there.